src/axelrod_dojo/archetypes/gambler.py

```diff
@@ -38,13 +43,19 @@ def __init__(self, plays, op_plays, op_start_plays, pattern=None):
 
     def player(self):
         """Turns the attribute vector in to a Gambler player instance."""
-        player = self.PlayerClass(pattern=self.vector)
+        parameters = Plays(self_plays=self.plays, op_plays=self.op_plays,
+                           op_openings=self.op_start_plays)
+        player = self.PlayerClass(pattern=self.pattern, parameters=parameters)
         return player
 
+    def copy(self):
+        return GamblerParams(plays=self.plays, op_plays=self.op_plays,
+                             op_start_plays=self.op_start_plays, pattern=self.pattern)
+
     def receive_vector(self, vector):
         """Receives a vector and creates an instance attribute called
         vector.  Ignores extra parameters."""
-        self.vector = vector
+        self.pattern = vector
```

Comment:
Can we change the parameter / doc string to be consistent with the use of pattern instead of parameter? Or not change the attribute name.